tutorials/install-cockroachdb-scaleway-instances/index.mdx

```diff
@@ -0,0 +1,480 @@
+    cp -i cockroach-v23.1.4.linux-amd64/lib/libgeos_c.so /usr/local/lib/cockroach/
+    ```
+5. Create the Cockroach directory.
+    ```
+    mkdir /var/lib/cockroach
+    ```
+6. Prepare and attach the data volume.
+    ```
+    mkfs -t ext4 /dev/sdb
+    e2label /dev/sdb DATA
+    echo "LABEL=DATA  	/var/lib/cockroach ext4 defaults 0 1" >> /etc/fstab
+    mount -a
+    ```
+7. Move the `certs` directory to the cockroach directory.
+    ```
+    mv certs /var/lib/cockroach/
+    ```
+8. Create a Unix user named `cockroach`.
+    ```
+    useradd cockroach
+    ```
+9. Change the ownership of the cockroach directory to the user cockroach.
+    ```
+    chown -R cockroach /var/lib/cockroach
+    ```
+10. Download the sample configuration template and save the file in the `/etc/systemd/system/` directory:
+    ```
+    curl -sSL -o /etc/systemd/system/cockroachdb.service https://raw.githubusercontent.com/cockroachdb/docs/master/_includes/v23.1/prod-deployment/securecockroachdb.service
+    ```
+
+    Alternatively, you can create the file yourself and copy the script into it:
+
+    ```
+    [Unit]
+    Description=Cockroach Database cluster node
+    Requires=network.target
+    [Service]
+    Type=notify
+    WorkingDirectory=/var/lib/cockroach
+    ExecStart=/usr/local/bin/cockroach start --certs-dir=certs --advertise-addr=<node1 address> --join=<node1 address>,<node2 address>,<node3 address> --cache=.25 --max-sql-memory=.25
+    TimeoutStopSec=300
+    Restart=always
+    RestartSec=10
+    StandardOutput=syslog
+    StandardError=syslog
+    SyslogIdentifier=cockroach
+    User=cockroach
+    [Install]
+    WantedBy=default.target
+    ```
+11. In the sample configuration template, specify values for the following flags:
+
+    | Flag | Description |
+    | :------------ | :------------------- |
+    | `--advertise-addr` | Specifies the IP address/hostname and port to tell other nodes to use. The port number can be omitted, in which case it defaults to `26257`. <br /><br /> This value must route to an IP address the node is listening on (with `--listen-addr` unspecified, the node listens on all IP addresses).<br /><br /> In some networking scenarios, you may need to use `--advertise-addr` and/or `--listen-addr` differently. For more details, see [Networking](https://www.cockroachlabs.com/docs/v23.1/recommended-production-settings#networking). |
+    | `--join`    | Identifies the address of 3 of the initial nodes of the cluster. These addresses should match the addresses that the target nodes are advertising. |
+
+    When deploying across multiple datacenters, or when there is otherwise high latency between nodes, it is recommended to set `--locality` as well, like for example `--locality=region=fr-par,zone=fr-par-1`. It is also required to use certain enterprise features. For more details, see [Locality](https://www.cockroachlabs.com/docs/v23.1/cockroach-start#locality).
```

Comment:
```suggestion
    When deploying across multiple data centers, or when there is otherwise high latency between nodes, it is recommended to set `--locality` as well, like for example `--locality=region=fr-par,zone=fr-par-1`. It is also required to use certain enterprise features. For more details, see [Locality](https://www.cockroachlabs.com/docs/v23.1/cockroach-start#locality).
```